package.json

```diff
@@ -31,6 +31,8 @@
   },
   "dependencies": {
     "clipboard": "2.0.10",
+    "favicons": "6.2.2",
```

Comment:
Please add it as a devDependency as it is only required during the build.

Reply:
Done